### ISSUE TITLE
Should have an easy way to update Mbed OS in exported project

### ISSUE THREAD

**Issue by janjongboom** [ARMmbed/mbed-cli#725]
I'm writing an application note for MDK and this is a real issue I encounter.

1. Find a project.
2. Export to MDK.
3. Now I want to update Mbed OS.

Because mbed-os is not under source control the normal `update`/`sync`/`deploy` things don't work. I need to:

1. Remove mbed and mbed-os folders.
2. Remove mbed.lib / mbed.bld / mbed-os.lib.
3. Run `mbed add mbed-os`.

Could we have some command that magically clears up all old Mbed folders and adds the latest?

Alternatively, a version of `mbed deploy` which will also work if the other folders are not under SCM could also work (but would need a big **WARNING** sign somewhere to avoid people losing data).

**Comment by ciarmcom** [comment]
ARM Internal Ref: MBOTRIAGE-1443

**Comment by theotherjimmy** [comment]
@janjongboom You're specifically talking about exports from the online compiler/os.mbed.com. Exports from Mbed CLI would just have to `mbed update` & `mbed export` again.

**Comment by janjongboom** [comment]
@theotherjimmy Yes, exports from Online Compiler. Now I need to have stuff in my documentation like:

```
Exported projects contain only a subset of Mbed OS 5 and cannot be managed by Mbed CLI. To resolve this:
5. Open a command prompt or terminal window, and navigate to the location where you unzipped the folder.
6. Run:
           rmdir mbed mbed-os /S
7. Then run:
mbed deploy
```

**Comment by theotherjimmy** [comment]
That is the workaround. If you have mbed-cli, what's the point of doing an export from the online compiler?

**Comment by janjongboom** [comment]
@theotherjimmy How else would I move from Online Compiler to MDK? This is not about importing existing examples, it's about migrating a codebase that I started online to MDK.

**Comment by theotherjimmy** [comment]
@janjongboom, That's a very good point. We should consider having a way to pull private examples in Mbed CLI, and fix this particular problem.

**Comment by ciarmcom** [comment]
Thank you for raising this issue. Please note we have updated our policies and 
 now only defects should be raised directly in GitHub. Going forward questions and 
 enhancements will be considered in our forums, https://forums.mbed.com/ . If this 
 issue is still relevant please re-raise it there.
 This GitHub issue will now be closed.

**Comment by adbridge** [comment]
Apologies, this was closed in error by the automated bot - re-opening.

**Comment by ciarmcom** [comment]
*Internal Jira reference: https://jira.arm.com/browse/IOTBTOOL-475*
